src/helix-ui/styles/modules/pagination.less

```diff
@@ -0,0 +1,19 @@
+.hxPagination {
+  background-color: @gray-0;
+  border: 1px solid @gray-300;
+  border-radius: 2px;
+
+  button {
+    border-color: transparent;
+    min-width: 2.50rem; //40px
```

Comment:
*NOTE*: (@araiford) this is not exactly to spec, but we found that it works better when transitioning between 1 and 2-digit page numbers.